bundles/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoPressureUnit.java

```diff
@@ -52,11 +53,11 @@ public static NetatmoPressureUnit fromString(String pressureUnit) {
 	
 	public BigDecimal convertPressure(BigDecimal value) {
 		if (this == DEFAULT_PRESSURE_UNIT) {
-			return value;
+			return value.setScale(0, RoundingMode.HALF_UP);
 		} else if (this == INHG) {
-			return value.multiply(MBAR_TO_INHG);
+			return value.multiply(MBAR_TO_INHG).setScale(2, RoundingMode.HALF_UP);
```

Comment:
This is well withing the range of accuracy of the device:  Accuracy: +-1 mbar / +- 0.03 inHg. https://www.netatmo.com/en-US/product/specifications/station

 I'll go back and review the ranges to make sure that I don't unnecessary remove accuracy.